CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * [#5077](https://github.com/bbatsov/rubocop/pull/5077): Add new `Rails/CreateTableWithTimestamps` cop. ([@wata727][])
 * Add new `Style/ColonMethodDefinition` cop. ([@rrosenblum][])
 * Add new `Style/ExtendSelf` cop. ([@drenmi][])
+* [#3437](https://github.com/bbatsov/rubocop/issues/3437): Add new `Lint/NumberConversion` cop. ([@albertpaulp][])
```

Comment:
This has to be rebased on top of the current `master` and moved the proper section of the changelog.

Reply:
I've rebased against current master. Up to date 👍